sass/_SmartSnippet.scss

```diff
@@ -1,13 +1,15 @@
 @import 'Variables';
 
-@mixin pointer($size, $offsetY) {
+@mixin pointer($size, $offsetX, $offsetY) {
   position: relative;
-  &::before {
-    content: '';
-    display: inline-block;
+  &::before,
+  &:before {
+    content: ' ';
     position: absolute;
-    top: $size / 2 + $offsetY;
-    @include triangle($size, $color-blue-background, up);
```

Comment:
Unfortunately, `triangle`'s borders wouldn't render on IE11.

Reply:
Would it be possible to reuse the `triangle` mixin and then add a single line adding a border?

Reply:
Well, I tested again with the original triangle and now it works?? I'll revert to using the `triangle` mixin.